mantidproject/mantid
# POWGEN and SNAP geometry bugfix

**Description of work.**

Add additional levels to the instrument tree for POWGEN to aid in creating different groupings. Also fix a small an error in the SNAP geometry.

**Report to:** POWGEN team, SNAP team, Marie Yao.

**To test:**

* Look at the tree in the instrument view for POWGEN.
* `CreateGroupingWorkspace` for POWGEN with "bank", "Column", and "Group"
* SNAP had a typo in detector ids for bank31. It should start at 393216 rather than 3933216.

*There is no associated issue.*

*This does not need to be in the release notes* because it is already covered by existing release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
